README.md

```diff
@@ -24,20 +24,18 @@ The software is considered to be at an **alpha** level of readiness--there are
 missing features and bugs to be fixed, but it is already used by many as a daily
 driver.
 
-Precompiled binaries are available for Windows through [appveyor](https://ci.appveyor.com/project/jwilm/alacritty).
-Precompiled binaries for other platforms are minimally blocked on a stable config format. For now, Alacritty must be
-built from source.
+Precompiled binaries are available from the [GitHub release page](https://github.com/jwilm/alacritty/releases).
```

Comment:
This should be changed to `GitHub releases page`, since the page is release**s**, not release. It could be considered to change it to `GitHub Releases page`, since it's the name of a GitHub feature, but I think both would be fine here.